Enhanced README.md (#1)
gluecose/gluecose: README.md

Complexity multiplied with a vast range of features amplify the trepidation.
Hence when a software developer wants to pick a COSE Library for their project,
one does not know, what one gets ?

> # Proposal

Author: Make it a Vision

> # Proposal

Reviewer: ```suggestion
# Vision
```

> * Have a clear view of tables that express the support of features by a certain COSE library, so that a user that needs to decide whether to use a certain crypto primitive for their software project can clearly identify the level of support for that feature in the open-source software ecosystem.

Reviewer 2: ```suggestion
Have a clear view of tables that express the support of features by a certain COSE library, so that a user that needs to decide whether to use a certain crypto primitive for their software project can clearly identify the level of support for that feature in the open-source software ecosystem.
```

# Deliverables

> 1. Feature support for a specific COSE implementation in an automated fashion.

Reviewer 2: ```suggestion
1. A way to query feature support for a specific COSE implementation in an automated fashion.
```

> 1. Feature support for a specific COSE implementation in an automated fashion.

Author: Knowledge about

> 2. Published results of the inter-operability tests passing for a specific COSE implementation.

Reviewer 2: ```suggestion
2. Published results of the glucose tests against a specific COSE implementation.
```

> 3. Overall Matrix of cross COSE compatibility with other libraies in the ecosystem.

Author: Fix typo

> 4. Validated Test Suites of COSE tests which can be used by developers in their projects.

Reviewer 2: ```suggestion
4. Comprehensive set of COSE tests which can be used by developers in their projects.
```

# Project Activites

> gluecose is a Work in Progress [WIP] project. Activities are tracked via github issues and progress tracked via  [gluecose project board](https://github.com/orgs/gluecose/projects/1)

Reviewer 2: ```suggestion
gluecose is an emerging project. Activities are tracked via the project [board](https://github.com/orgs/gluecose/projects/1).
```
